pkgs/misc/emulators/yuzu-mainline/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, mkDerivation, fetchgit, cmake, libressl, SDL2, qtbase, python2 }:
+
+mkDerivation rec {
+  pname = "yuzu-mainline";
+  version = "unstable-2020-04-13";
+
+  # Submodules
+  src = fetchgit {
+    url = "https://github.com/yuzu-emu/yuzu-mainline";
+    rev = "20c42dc9ea8bdc208d9d274350af2ff347876c76";
+    sha256 = "0s9p5di9kcr8sh6iipv21b74c4dhn37an4khy4y7b9s9y8p1haim";
+  };
+
+  # Trick the configure system
+  preConfigure = ''
+    sed -n 's,^ *path = \(.*\),\1,p' .gitmodules | while read path; do
+      mkdir "$path/.git"
+    done
+  '';
+
+  enableParallelBuilding = true;
```

Comment:
I believe you don't need to specify `enableParallelBuilding` in this instance, from the nixpkgs manual:

> Unless set to false, some build systems with good support for parallel building including cmake, meson, and qmake will set it to true. 

So you don't need to specify this flag at all in your packages, unless to disable it. It's already on by default when it ought to be, and can be disabled when it causes problems.